cli/src/commands/start.rs

```diff
@@ -137,6 +137,9 @@ pub struct Start {
     /// If development mode is enabled, specify the number of genesis validators (default: 4)
     #[clap(long)]
     pub dev_num_validators: Option<u16>,
+    /// If developtment mode is enabled, specify whether node 0 should generate traffic to drive the network
+    #[clap(long = "dev-traffic")]
+    pub dev_traffic: bool,
```

Comment:
Is this `true` by default?

Reply:
It is `false` by default, made it explicit: d3cc72bbd

I realise this *may* lead to issues in the community because nodes require this flag to auto-advance blocks. But I think it is the right default for testing and the updated README 281acb5d3, `devnet.sh` should be sufficient guidance.

Let me know if you prefer an alternative `dev-no-traffic` command.

Reply:
It should be true by default.

Can use `no-dev-txs` as the flag name. Traffic implies network traffic among clients/validators.

Reply:
Done

de2bd055c

8a0145c35